lc2fen/predict_board.py

```diff
@@ -426,21 +441,26 @@ class as another list.
     return fen
 
 
-def print_fen_comparison(board_name, fen, correct_fen):
+def print_fen_comparison(board_name, fen, correct_fen, used_previous_fen):
     """
     Compares the predicted fen with the correct fen and pretty prints
     the result.
 
     :param board_name: Name of the board. For example: 'test1.jpg'
     :param fen: Predicted fen string.
     :param correct_fen: Correct fen string.
+    :param used_previous_fen: Whether previous fen was used during prediction.
     """
     n_dif = compare_fen(fen, correct_fen)
+    used_previous_fen_str = (
+        "_with_previous_fen" if used_previous_fen else "_without_previous_fen"
+    )
     print(
         board_name[:-4]
+        + used_previous_fen_str
         + " - Err:"
         + str(n_dif)
-        + " Acc:{:.2f}% FEN:".format(1 - (n_dif / 64))
```

Comment:
This was a bug that I didn't notice until I worked on updating the test functions; if we want to show the "%" sign, we must multiply the result by 100